pkg/systemd/services.css

```diff
@@ -118,3 +118,7 @@ table.systemd-unit-relationship-table td:first-child {
 #services .nav-tabs {
     border-bottom: none;
 }
+
+.nav-tabs li {
+  outline: none;
```

Comment:
We _definitely_ want an outline of the focused element somewhere, BTW. So if we remove it here, it needs to be done elsewhere...

See:
- https://a11yproject.com/posts/never-remove-css-outlines/
- https://css-tricks.com/removing-the-dotted-outline/